website/src/pages/pricing.tsx

```diff
@@ -30,80 +34,71 @@ export default ((props: any) => (
                 <link rel="icon" type="image/png" href="https://about.sourcegraph.com/favicon.png" />
             </Helmet>
             <div className="pricing-page">
+                <ContentSection color="white" className="hero-section text-center py-5">
+                    <h4 className="font-weight-light">
+                        Sourcegraph code search, code navigation,&nbsp;
+                        <PricingFreeTierPopoverButton />
+                        are free for individuals and small teams (up to 20 users). <a href="https://docs.sourcegraph.com">Install</a> to get started today!
+                    </h4>
+                </ContentSection>
                 <div className="container-fluid pricing-page__plans">
                     <div className="row pt-6">
                         <div className="col-8 col-md-4 mx-auto mb-4">
                             <PricingPlan
                                 className="pricing__plan"
-                                name="Core"
-                                description="Helping developers search, browse, and grok their code, for faster software development and incident response."
-                                price="$0"
-                                priceInterval=""
-                                priceCaption={
-                                    <>
-                                        for individuals
-                                        <br />
-                                        and small teams
-                                    </>
-                                }
+                                name="Enterprise"
+                                description="Enabling engineering and DevOps leaders to speed up the organization's software development and monitor risks."
+                                price="$29"
                                 features={[
-                                    { name: 'Code search', id: 'code-search' },
-                                    { name: 'Code navigation (definitions & references)', id: 'code-navigation' },
-                                    { name: 'Sourcegraph extensions', id: 'integrations' },
-                                    { name: 'Code host & editor integration', id: 'integrations' },
+                                    { name: 'Code review & pull request integration', id: 'code-review' },
+                                    { name: 'Code alerts & automation', id: 'code-alerts-automation' },
```

Comment:
We should be careful about using 'automation' here if we are going to use that as the name for large scale code mod which we want in the top tier.